rosdep/base.yaml

```diff
@@ -2336,10 +2337,13 @@ libqglviewer-qt4:
     vivid: [libqglviewer2]
     wily: [libqglviewer2-qt4]
     xenial: [libqglviewer2-qt4]
+    yakkety: [libqglviewer2-qt4]
+    zesty: [libqglviewer2-qt4]
 libqglviewer-qt4-dev:
   arch: [libqglviewer-qt4]
   debian:
     jessie: [libqglviewer-dev]
+    stretch: [libqglviewer-qt4-dev]
```

Comment:
@ahornung looks like this key is `libqglviewer-dev-qt4` on jessie, I think that's the reason why http://build.ros.org/job/Lbin_ds_dS64__octovis__debian_stretch_amd64__binary is failing